### ISSUE TITLE
Display X page by itself

### ISSUE THREAD

**Issue by zzyzzyzzx** [BookStackApp/BookStack#370]
Not sure what this would be considered, more of a support i guess.

Anyways, I am looking for a way to display a page just all on its own, like when someone visits my website i would like them to get information from a page, then click on a link and another page pops up, I have a custom layout that took me longer than i care to admit to work, and its not even done, it still has a ton of errors but thats moot.

to recap i basically want to be able to go into a book->chapter->page and display that from certain links (pretty sure i can just do a direct url to it). second thing was showing a page when a person goes to the website so it always shows X page instead of the most recent thing

**Comment by Shackelford-Arden** [comment]
@zzyzzyzzx What do you mean by

> get information from a page, then click on a link and another page pops up 

Do you mean you want to have this new page open in a new tab/window?

As for your second item, it sounds like you want to be able to set a custom home page. Is that right?

**Comment by zzyzzyzzx** [comment]
took me a minute to get it up and running, its still busted but this is what i mean:

http://hadar.ninja/TestDirectory


that was my old "wiki style" website basically where i made/stored all the info in a database and called on it to make the content of the pages,  but moving to laravel i did not want to write my own wiki again and i found this one, so i wanted to emulate what i had

basically a few static pages, that can be "wiki" then a backend that can only be edited by specific people

**Comment by Shackelford-Arden** [comment]
Hmm. I'm not 100% sure I understand what your end goal is, but I would think that you could edit the permissions for books/chapters/pages and have it be where "visitors" have access to some pages, but the rest are hidden unless you're logged in.

**Comment by zzyzzyzzx** [comment]
I can i figured that out

but its like that page, I just want to show a static page when everyone goes to url (like my last url)

**Comment by Shackelford-Arden** [comment]
This sounds a lot like #126 . There are many of us who want to be able to set a custom home page.

**Comment by zzyzzyzzx** [comment]
it is thank you, closing this, i thought i searched well enough but i guess not, anyways i guess ill look through the code for a hack
